.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
Should we have these in eslint?

Reply:
see my comment [here](https://github.com/forcedotcom/lwc-dev-mobile-core/pull/83#discussion_r1599216704).